src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
Add empty lines between multiline sibling blocks of HTML. In this case, an empty line should be added between the <head> and <body> sections to improve readability.